### DISCUSSION TITLE
Google AMP Support

### DISCUSSION THREAD

**Discussion by ghost** [facebook/docusaurus#4043] [Ideas]
like nextjs

**Comment by slorber** [comment]
How would you expect this to work?

What I see is that Next provides AMP support but it's not automatic, you have to actually do modifications on it and have 2 distinct rendering logics for html/amp for each page
https://nextjs.org/docs/api-reference/next/amp

Do you expect this to work for all pages of the site? For React pages? For Markdown docs?

A Docusaurus site is mostly markdown/MDX docs, how would you render such a doc to produce an AMP output? How Next does this (if it even does this?). How do we convert non-std, interactive elements to AMP? (such as tabs, admonitions?)
